boards/shields/nxp_btb44_ov5640/nxp_btb44_ov5640.overlay

```diff
@@ -4,6 +4,8 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
+#include <zephyr/dt-bindings/video/video-interfaces.h>
```

Comment:
Same thing here, these test changes need to be in the same commit as the binding changes